locales/templates/app.pot

```diff
@@ -1405,9 +1481,140 @@ msgid "Rather see the full desktop site?%1$s Menu > Request desktop site"
 msgstr ""
 
 #. Tip displayed on home view explaining how to add a custom autocomplete URL
-#. Argument 1 is the app name (Firefox Focus/Firefox Klar/etc).
+msgctxt "tip_disable_tips2"
+msgid "Turn off tips on the start screen"
+msgstr ""
+
+#. Displayed in a snackbar when a user opens a new tab, gives action to switch
+#. to the newly opened tab
+msgctxt "new_tab_opened_snackbar"
+msgid "New tab opened"
+msgstr ""
+
+msgctxt "open_new_tab_snackbar"
+msgid "Switch"
+msgstr ""
+
+#. Preference for switching to a new tab immediately after opening
+msgctxt "preference_open_new_tab"
+msgid "Switch to link in new tab immediately"
+msgstr ""
+
+#. deprecated strings
 #, c-format
 msgctxt "tip_disable_tips"
 msgid "Turn off tips on the %1$s start screen"
 msgstr ""
 
+msgctxt "preference_switch_autocomplete"
+msgid "Autocomplete"
+msgstr ""
+
+msgctxt "preference_category_autocomplete_preinstalled"
+msgid "Default URL List"
+msgstr ""
+
+#, c-format
+msgctxt "preference_autocomplete_preinstalled_summary"
```

Comment:
Needs localization comment explaining what "enable" is referring to

Reply:
Same as above